core/src/test/java/org/hibernate/ogm/backendtck/associations/collection/types/ListTest.java

```diff
@@ -19,162 +18,130 @@
  */
 public class ListTest extends OgmTestCase {
 
+	Child luke = new Child( "Luke" );
+	Child leia = new Child( "Leia" );
+	Father father = new Father();
+	Race race = new Race();
+
+	GrandMother grandMother = new GrandMother();
+
+	@After
+	public void after() {
+		delete( race.getRunnersByArrival().toArray() );
+		delete( luke, leia, father, grandMother, race );
+		checkCleanCache();
+	}
+
+	private void delete(Object... entities) {
```

Comment:
I prefer to wait, we already have the `deleteAll` but because this test is older and uses several different entities that method is not right.

If in the future we need something similar again, we can extract it.

Reply:
Ok. Then I'm going to merge it.